src/Illuminate/Support/Debug/Dumper.php

```diff
@@ -15,6 +15,12 @@ class Dumper
      */
     public function dump($value)
     {
+        if (! class_exists(CliDumper::class)) {
+            var_dump($value);
+
+            return;
```

Comment:
I think Taylor would prefer:

```php
if (class_exists(CliDumper::class)) {
    $dumper = 'cli' === PHP_SAPI ? new CliDumper : new HtmlDumper;

    $dumper->dump((new VarCloner)->cloneVar($value));
} else {
    var_dump($value);
}
```

over an early return from an if statement in a context where we really do have else if.